e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_1_spec.js

```diff
@@ -441,6 +441,38 @@ describe('Keyboard Shortcuts', () => {
         cy.get('#moreDmModal').should('not.exist');
     });
 
+    it('MM-T4452 - CTRL/CMD+SHIFT+. Expand or collapse RHS', () => {
+        // # Post a message and open RHS
+        const message = `hello${Date.now()}`;
+        cy.postMessage(message);
+        cy.getLastPostId().then((postId) => {
+            // # Mouseover the post and click post comment icon.
+            cy.clickPostCommentIcon(postId);
+        });
+
+        // # Type CTRL/CMD+SHIFT+. to expand 'RHS'
+        cy.get('#post_textbox').cmdOrCtrlShortcut('{shift}.');
+        cy.uiGetRHS().isExpanded();
+
+        // # Type CTRL/CMD+SHIFT+. to collapse 'RHS'
+        cy.get('#post_textbox').cmdOrCtrlShortcut('{shift}.');
+        cy.uiGetRHS();
```

Comment:
```suggestion
        // * Verify RHS is now in collapsed state
        cy.get('#sidebar-right').should('be.visible').and('not.have.class', 'sidebar--right--expanded');
```
`cy.uiGetRHS()` only tests if RHS is visible, but here we explicitly need to verify that its in collapsed that hence the above